README.md

```diff
@@ -6,36 +6,36 @@ ExAudit plugs right into your ecto repositories and hooks all the data mutating
 to track changes to entities in your database.
 
 ## Features
- 
- * Wraps Ecto.Repo, no need to change your existing codebase to start tracking changes
- * Creates +- diffs of the casted structs. Custom types are automatically supported.
- * Ships with functions to review the history of a struct and roll back changes
- * Allows custom ID types and custom fields in the version schema
- * Tracks associated entities when they're created, updated or deleted in a single Repo call
- * Recursively tracks cascading deletions
+
+- Wraps Ecto.Repo, no need to change your existing codebase to start tracking changes
```

Comment:
My IDE did this changes (- instead of *) automatically when I saved the file. I hope you don't mind.

Reply:
All this reformating makes it really hard to see what you actually changed.

Reply:
Sorry, I don't know why my VS Code formatter changed those.